__tests__/app.test.js

```diff
@@ -81,3 +81,32 @@ describe("GET /api/reviews/:review_id", () => {
             });
     });
 });
+
+describe("GET /api/reviews", () => {
+    test("200: Responds with an array of review objects", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        review_body: expect.any(String),
+                        designer: expect.any(String),
+                        review_img_url: expect.any(String),
+                        votes: expect.any(Number),
+                        category: expect.any(String),
+                        owner: expect.any(String),
+                        comment_count: expect.any(Number),
+                    });
+                });
+                const reviewsCopy = [...reviews];
+                const sortedReviews = reviewsCopy.sort((reviewA, reviewB) => {
```

Comment:
Nice use of .sort(). I would look at refactoring to use jest-sorted here, just because it's slightly easier to write and read. I'd also suggest moving this to its own test case to separate our concerns (e.g. do we get all reviews back, are they how we expect them to look, and are they returned in order).